src/components/cards/TradeCard/TradeRoute.vue

```diff
@@ -433,9 +434,9 @@ export default defineComponent({
     function getPoolLink(id: string): string {
       const chainId = appNetworkConfig.chainId;
       const prefixMap = {
-        1: 'app.',
-        42: 'kovan.',
-        137: 'polygon.'
+        [Network.MAINNET]: 'app.',
+        [Network.KOVAN]: 'kovan.',
+        [Network.POLYGON]: 'polygon.'
```

Comment:
Throughout this refactor, I'll replace all the numeric/string network ids to use the `Network` enum for better readability.